src/platform/plugins/shared/workflows_management/public/widgets/workflow_yaml_editor/styles/get_monaco_workflow_overrides_styles.ts

```diff
@@ -26,6 +26,10 @@ export const getMonacoWorkflowOverridesStyles = (euiThemeContext: UseEuiTheme) =
       max-height: 400px;
       font-size: 13px;
       z-index: 999;
+      border-radius: 6px !important;
```

Comment:
can theoretically be
```suggestion
      border-radius: ${euiTheme.border.radius.medium} !important;
```
but @r4zr32d3k1l designs declares 6px, and not 4px.